website/versioned_docs/version-5.x/ecosystem-angular.md

```diff
@@ -499,6 +499,37 @@ Then use it in your template:
 ```html
 <img [src]="'yoshi.png' | assetUrl" />
 ```
+##### Option 3
+
+upload via sass; This demonstration was done with the angular 8 and 9 versions, example: 
```

Comment:
What does "upload via sass" mean? This appears to be in error.

The first sentence of this option should describe the overall approach.

```suggestion
You can use Angular's [rebaseRootRelativeCssUrls](https://angular.io/cli/build) option to rewrite the URLs. This option has been deprecated in the latest versions of Angular.
```

Reply:
Hello, well first of all, sorry, I am Peruvian and I still don't speak English very well; what i meant is "load an image with scss"

Reply:
No worries at all - I am glad you're contributing and can help out with any minor grammatical errors. This approach is not specific to Sass, though, right? It also works with Less and CSS?

Reply:
Test with sass and css and it works, I guess it applies for Less

Reply:
👍 let's not make the documentation refer specifically to Sass then.